cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,51 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('Settings page', () => {
-  before(() => {
 
-  });
+import SignInPageObject from '../support/pages/signIn.pageObject';
+import homePageObject from '../support/pages/home.pageObject';
+import SettingsPageObject from '../support/pages/settings.pageObject';
 
-  beforeEach(() => {
+const updateSettings = new SettingsPageObject();
+const signInPage = new SignInPageObject();
+const homePage = new homePageObject();
 
+describe('Settings page', () => {
+  let user;
+  let bio = 'zhazam';
+  beforeEach(() => {
+    cy.task('db:clear');
+    cy.task('generateUser')
+      .then(generateUser, () => {
```

Comment:
can't run your test file. I have error 
![image](https://github.com/mate-academy/qa_cypress_e2e_settings_pom_local/assets/85580286/35333412-183b-49cf-b09d-b34995c48068)

Reply:
Linter made me do this